src/connectors/naxosmusiclibrary.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const playerBar = '#music';
+
+function getCatalogID() {
+	return Util.getTextFromSelectors(`${playerBar} .u-album`);
+}
+
+Connector.useMediaSessionApi();
+
+Connector.playerSelector = '.u-control.u-process';
+
+Connector.artistSelector = `${playerBar} .u-artist`;
+
+Connector.trackSelector = `${playerBar} .u-music-title`;
+
+Connector.getAlbum = () => {
+	const catalogID = getCatalogID();
+	const albumHeader = Util.queryElements(`.eps._playAlbum[data-catalogueid="${catalogID}"]`);
+	if (albumHeader.length > 0) {
+		return albumHeader[0].textContent;
+	} {
```

Comment:
We can drop the `{` + the closing `}` since we're returning in the `if.

Aside that we'c an merge this 👍🏼

Reply:
Linter insists on braces around the return, but I dropped the other pair of braces.